packages/mdc-dialog/test/component.test.ts

```diff
@@ -591,4 +591,35 @@ describe('MDCDialog', () => {
     component.layout();
     expect((component as any).foundation.layout).toHaveBeenCalled();
   });
+
+  it(`Button with ${strings.INITIAL_FOCUS_ATTRIBUTE} will be focused when the dialog is opened, with multiple initial focus buttons in DOM`, () => {
+    const {root: root1, component: component1, yesButton: yesButton1} = setupTest();
+    const {root: root2, component: component2, yesButton: yesButton2} = setupTest();
+
+    const initialFocusEl1 = (component1.getDefaultFoundation() as any).adapter.getInitialFocusEl();
+    expect(initialFocusEl1).toEqual(yesButton1);
+
+    const initialFocusEl2 = (component2.getDefaultFoundation() as any).adapter.getInitialFocusEl();
+    expect(initialFocusEl2).toEqual(yesButton2);
```

Comment:
Can remove these. Let's avoid accessing adapter methods that should be internal to foundation.

We can replace this with asserting that the button elements have the INITIAL_FOCUS_ATTRIBUTE attr's: https://github.com/material-components/material-components-web/blob/e590b376bf20bde50e6f6b62339c0bac2703ccf0/packages/mdc-dialog/constants.ts#L43